h/groups/models.py

```diff
@@ -34,9 +34,9 @@ class Group(Base, mixins.Timestamps):
     creator = sa.orm.relationship('User')
 
     # Group membership
-    members = sa.orm.relationship('User',
-                                  secondary='user_group',
-                                  backref='groups')
+    members = sa.orm.relationship(
+        'User', secondary='user_group', backref=sa.orm.backref(
+            'groups', order_by=sa.func.lower(name)))
```

Comment:
that's what I wanted to do first, but I'm getting this: `NameError: name 'Group' is not defined`